Exercise_6/ReversePolishNotation.js

```diff
@@ -0,0 +1,44 @@
+// const process = require('process');
+// console.log(calculate(process.argv[2]));
+
+function calculate(argument,extraOperators) {
+    const invalidArgumentError = new Error('The argument ' + argument + ' is invalid');
+    const signs = argument.split(' ');
+    const stack=[];
+
+    function isNumber(sign) {
+        return !isNaN(parseInt(sign)) && isFinite(sign);
+    }
+
+    function isValidOperator(sign) {
```

Comment:
You still don't have the ability to divide. And I don't like the idea of the function. See the comment about my thoughts below.